doc/tag_filter.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python2
-
```

Comment:
No need to remove the shebang.  The previous comment only about the missing @PYTHON_EXECUTABLE@ in the Doxyfile.in, because we had problems with different python executable versions used in the same build once.  If the shebang helps you with testing feel free to readd it, but that's your call.